route/route.go

```diff
@@ -215,7 +222,7 @@ func (r *Route) weighTargets() {
 	}
 
 	// compute the weight for the targets with dynamic weights
-	dynamic := (1 - sumFixed) / float64(len(r.Targets)-nFixed)
+	dynamic := float64(1-sumFixed) / float64(len(r.Targets)-nFixed)
```

Comment:
gofmt?

Reply:
If you divide something by a `float64` then the result can never be an `int` since Go doesn't auto-convert between numeric types. `const` values are the only exception.

Reply:
Ok, this makes sense. My bad.

Reply:
No worries. You live and learn :)